roles/httpd.lua

```diff
@@ -107,6 +107,20 @@ local function parse_params(node)
     }
 end
 
+local function server_was_changed(name, node, host, port)
```

Comment:
I'm not sure, but isn't it
```suggestion
local function server_has_changed(name, node, host, port)
```